doc/release-notes/9667-metadatablocks-api-extension.md

```diff
@@ -0,0 +1 @@
+DatasetFieldType attribute "displayFormat", is now returned by the API.
```

Comment:
Should reference the API call involved and make it clearer that goes in the  backward compatibility section